src/Components/NotificationItem.vue

```diff
@@ -10,12 +10,11 @@
 		:data-object-type="objectType"
 		:data-app="app">
 		<div class="notification-heading">
-			<span class="hidden-visually">{{ absoluteDate }}</span>
-			<span
+			<NcDateTime
```

Comment:
Similar to https://github.com/nextcloud/announcementcenter/pull/945/files#r2158705117 I'd also here use a bit longer format on the date for human-readability
@Antreesy did you notice the change and think the new format is good/better? Or would you be okay reverting to the previous longer format?

Reply:
I personally like short version, if it's about `20. Dezember 2020` against `20.12.2020`, 

But we can pass `:format="{ dateStyle: 'long' }"` to keep the previous formatting

Reply:
@nickvergessen in that case I would say designers should be asked and we change the default on `@nextcloud/vue` as this is also the same on files app, logreader etc.
Meaning we should have a consistent design on that.